python/lsst/sims/maf/metrics/summaryMetrics.py

```diff
@@ -224,3 +226,90 @@ def run(self, dataSlice, slicePoint=None):
         condition = np.where((ell <= self.lmax) & (ell >= self.lmin))[0]
         totalpower = np.sum(cl[condition]*(2*ell[condition]+1))
         return totalpower
+
+
+
+class FoMEmulatorMetric(BaseMetric):
```

Comment:
Is there a more specific name you could make for this? I get that FoMEmulator make sense for your figure of merit, but there are many figures of merit that we need to consider. Maybe WeakLensing_FoM_Emulator or something similar?